tests/main/classic-firstboot/task.yaml

```diff
@@ -62,6 +62,9 @@ execute: |
         sleep 1
     done
 
+    echo "Ensure snapd.seeded.service is active"
+    systemctl status snapd.seeded.service | MATCH "SUCCESS"
```

Comment:
You could just do `if ! systemctl is-active snapd.seeded.service; then ...`. It returns 0 on success...

Reply:
IIRC just `systemctl status <foo>.service` returns non-0 if the service has failed.

Reply:
This is not as useful as it looks, it gives non-zero (3) for success case like this:
```
● snapd.seeded.service - Wait until snapd is fully seeded
   Loaded: loaded (/lib/systemd/system/snapd.seeded.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Thu 2018-06-21 07:58:54 UTC; 17min ago
 Main PID: 12302 (code=exited, status=0/SUCCESS)

Jun 21 07:58:34 jun210949-964463 systemd[1]: Starting Wait until snapd is fully seeded...
Jun 21 07:58:34 jun210949-964463 systemd[1]: Started Wait until snapd is fully seeded.
Jun 21 07:58:54 jun210949-964463 systemd[1]: Stopped Wait until snapd is fully seeded.

```
but I also saw:
```
● snapd.seeded.service - Wait until snapd is fully seeded
   Loaded: loaded (/lib/systemd/system/snapd.seeded.service; enabled; vendor preset: enabled)
   Active: active (exited) since Mon 2018-06-18 09:02:45 CEST; 3 days ago
 Main PID: 1020 (code=exited, status=0/SUCCESS)
    Tasks: 0 (limit: 4915)
   CGroup: /system.slice/snapd.seeded.service
```
and it gives status 0. In both cases the actual process exited with 0.

Reply:
That's probably because we have `RemainAfterExit=true` and tthe service was stopped as in `systemctl stop snapd.seeded.service` in prepare? I think we actually want it to be in active state and check that the socket activation works. In which case, instead of `systemctl start snapd.service snapd.socket` you'd want to do `systemctl start snapd.seeded.service` which should autostart the socket and thus snapd.